tests/rkt_fetch_test.go

```diff
@@ -147,10 +147,6 @@ func TestImplicitFetch(t *testing.T) {
 			t.Fatalf("Cannot exec rkt: %v", err)
 		}
 
-		if err := child.Expect(foundMsg); err != nil {
-			t.Fatalf("Expected %q but not found: %v", foundMsg, err)
-		}
-
```

Comment:
Yes. I can only think of that need to force download of an updated images with the same url using a server returning a big max age (but that's a server configuration problem). If really needed it can be implemented after this, it was just a way to trying fixing the check without removing that part.

Should I just remove/disable that check?
I tried to use coreos.com/etcd ACIs as the hosting server provides max-age and ETag but they doesn't have latest images but only versioned ones.